app/models.py

```diff
@@ -762,6 +764,14 @@ def __init__(self, **kwargs):
     archived = db.Column(db.Boolean, nullable=False, default=False)
     hidden = db.Column(db.Boolean, nullable=False, default=False)
     subject = db.Column(db.Text)
+    postage = db.Column(db.String, nullable=True)
+    CheckConstraint("""
+        CASE WHEN template_type = 'letter' THEN
+            postage in ('first', 'second', null)
```

Comment:
this won't work like you would expect, because null doesn't equal null. (see `nan poisoning`). The only operation you should do with null is `is` or `is not`

```
notification_api=# select null is null;
 ?column?
----------
 t
(1 row)

notification_api=# select null in (null);
 ?column?
----------

(1 row)
```

Reply:
Does this need to be a check constraint or just a typed column?

Reply:
I'm not sure, I followed pattern from class NotificationBase where postage column has a check constraint...